pallets/parachain-system/src/lib.rs

```diff
@@ -155,6 +155,19 @@ decl_module! {
 			Self::schedule_upgrade_impl(validation_function)?;
```

Comment:
Whats going on up here?

`schedule_upgrade` and `schedule_upgrade_without_checks` are exactly the same?

Reply:
https://github.com/paritytech/cumulus/issues/374